source/_posts/2020/2020-01-27-switch-travis-to-github-actions-to-reduce-stress.md

```diff
@@ -226,7 +225,7 @@ How to work around this? If we create a new organization that is **open-source o
 
 **In Travis**, you could add a badge for the whole build on a specific branch. It was nice that it skipped allowed failure.
 
-**In GitHub Actions**, it's different. ~~The badge covers **1 specific workflow**. This workflow should contain all the relevant jobs~~.
+**In GitHub Actions**, it's different. [~~The badge covers **1 specific workflow**.] This workflow should contain all the relevant jobs~~.
```

Comment:
I think ~~....~~ cannot be separated by [], it will break the markdown.

How does this look locally? What is this change for?

Reply:
Rollbacked - fixing only typos now.